native/cocos/bindings/auto/jsb_cocos_auto.cpp

```diff
@@ -4616,7 +4616,7 @@ static bool js_cc_ISystemWindowManager_processEvent(se::State& s)
     
     ok &= sevalue_to_native(args[0], &arg2, s.thisObject());
     SE_PRECONDITION2(ok, false, "Error processing arguments"); 
-    (arg1)->processEvent(arg2);
+    (arg1)->processEvent();
```

Comment:
arg2 is still defined. Did you re-generate the `auto binding code`?

Reply:
Will this be automatically overwritten? Shouldn't this file theoretically be submitted?

Reply:
Yes ,it will be overrode by CocosRobot, but that need to send a PR. You could commit this file since the file generated by swig is consistent.